packages/spruce-node/index.ts

```diff
@@ -687,40 +737,65 @@ export default class Sprucebot {
 		return this.adapter.delete(`/data/${id}`)
 	}
 
-	/**
-	 * Emit a custom event. The response is the response from all skills
-	 */
+	/** Emit a custom event. The response is the response from all skills */
 	public async emit(
+		/** the location that will receive this event */
 		locationId: string,
+		/** the name of the event */
 		eventName: string,
+		/** any data you want passed with the event */
 		payload: Record<string, any> = {},
-		options?: Record<string, any>,
+		/** options to control how the event behaves */
+		options?: IEmitEventOptions,
+		/** DEPRECATED, please pass options.eventId */
 		eventId?: string
 	): Promise<IEventResponse[]> {
+		let actualEventId = eventId
+		if (options && options.eventId) {
+			actualEventId = options.eventId
+		}
+
+		if (eventId) {
+			debug(
```

Comment:
We should probably use `log.warn` here so the skills dev sees it without needing to enable debug